tests/readmes/test_dispenser_flow.py

```diff
@@ -0,0 +1,88 @@
+#
+# Copyright 2021 Ocean Protocol Foundation
+# SPDX-License-Identifier: Apache-2.0
+#
+import os
+
+from ocean_lib.example_config import ExampleConfig
+from ocean_lib.models.dispenser import Dispenser
+from ocean_lib.models.models_structures import ErcCreateData, DispenserData
+from ocean_lib.ocean.mint_fake_ocean import mint_fake_OCEAN
+from ocean_lib.ocean.ocean import Ocean
+from ocean_lib.ocean.util import get_address_of_type
+from ocean_lib.web3_internal.constants import ZERO_ADDRESS
+from ocean_lib.web3_internal.currency import to_wei
+from ocean_lib.web3_internal.wallet import Wallet
+
+
+def test_dispenser_flow():
+    config = ExampleConfig.get_config()
+    ocean = Ocean(config)
+
+    # Create Alice's wallet
+    alice_private_key = os.getenv("TEST_PRIVATE_KEY1")
+    alice_wallet = Wallet(
+        ocean.web3,
+        alice_private_key,
+        config.block_confirmations,
+        config.transaction_timeout,
+    )
+    assert alice_wallet.address
+
+    # Mint OCEAN
+    mint_fake_OCEAN(config)
+    assert alice_wallet.web3.eth.get_balance(alice_wallet.address) > 0, "need ETH"
+
+    # Publish an NFT token
+    erc721_token = ocean.create_nft_token("NFTToken1", "NFT1", alice_wallet)
+    token_address = erc721_token.address
+    assert token_address
+    assert erc721_token.token_name() == "NFTToken1"
+    assert erc721_token.symbol() == "NFT1"
+
+    # Prepare data for ERC20 token
+    cap = to_wei(100)
+    erc20_data = ErcCreateData(
```

Comment:
This may not be the place to ask, but why is there a class with "erc" as the prefix? ERC should always refer to _some_ specific standard, eg erc721 or erc20. And there are literally thousands more.

Reply:
That's pretty vague. How about `createErc20Data`? That aligns with the resulting variable `erc20_data`.

Reply:
Ok and for the NFT, I used `NftCreateData`, I'll switch to `CreateErc721Data`.

Reply:
I agree with Trent on this one. I thought about it myself but I was a bit caught up with bigger fish :) However, I think it should be a separate task and a separate PR. If you agree, @trentmc, I will proceed with reviewing the rest of the PR. If not, I will wait for the naming part.

Reply:
FYI @mariacarmina made a new issue: https://github.com/oceanprotocol/ocean.py/issues/660

Therefore you can proceed with reviewing the rest of the PR